libs/pages/services/src/lib/feature/page-helm-create-feature/step-values-override-files-feature/step-values-override-files-feature.tsx

```diff
@@ -0,0 +1,277 @@
+          <Heading className="mb-2">Values override as file</Heading>
+          <p className="text-sm text-neutral-350 mb-2">
+            Define the YAML file(s) to be applied as override to the default values.yaml delivered with the chart. It is
+            highly recommended to store the override file(s) in a git repository.
+          </p>
+          <Popover.Root>
+            <Popover.Trigger>
+              <span className="text-sm cursor-pointer text-brand-500 hover:text-brand-600 transition font-medium mb-5">
+                How it works <Icon className="text-xs" name={IconAwesomeEnum.CIRCLE_QUESTION} />
+              </span>
+            </Popover.Trigger>
+            <Popover.Content side="left" className="text-neutral-350 text-sm relative" style={{ width: 440 }}>
+              <h6 className="text-neutral-400 font-medium mb-2">How it works</h6>
+              <ul className="list-disc pl-4">
+                <li>
+                  You can specify one or more YAML file to be used to override the values.yaml delivered with the chart.
+                  These will be passed via the -f helm argument.
+                </li>
+                <li>
+                  The files can be retrieved either from a git repository (preferred option) or they can be stored as
+                  raw YAML by Qovery (no history is retained). The chosen git repository can be a repository different
+                  from the one of the chart.
+                </li>
+                <li>
+                  If you don’t have a file, you can skip this step and instead define the values override directly as
+                  arguments (--set).
+                </li>
+                <li>
+                  You can use the Qovery environment variables as overrides by using the pattern
+                  “qovery.env.ENV_VAR_NAME”.
+                </li>
+                <li>
+                  To get all the Qovery functionalities, add the macro “qovery.labels.service” within the field managing
+                  the labels assigned to the deployed pods.
+                </li>
+              </ul>
+              <Popover.Close className="absolute top-4 right-4">
+                <button type="button">
+                  <Icon name={IconAwesomeEnum.XMARK} className="text-lg leading-4 text-neutral-400" />
+                </button>
+              </Popover.Close>
+            </Popover.Content>
+          </Popover.Root>
+          <Button
+            size="lg"
+            variant="surface"
+            color="neutral"
+            className="mb-10"
+            loading={isLoadingHelmDefaultValues}
+            onClick={() => createHelmDefaultValuesMutation()}
+          >
+            See default values.yaml <Icon className="text-xs ml-2" name={IconAwesomeEnum.ARROW_UP_RIGHT_FROM_SQUARE} />
+          </Button>
+          <form onSubmit={onSubmit} className="w-full">
+            {/* TODO: should be update with TabsPrimitive component */}
```

Comment:
:+1: should be discussed with design team

Reply:
Yes done, it's a Select and not a Tabs now I'll replace it !